src/platform_impl/windows/event_loop.rs

```diff
@@ -2101,7 +2101,8 @@ unsafe extern "system" fn thread_event_target_callback<T: 'static>(
                     }
 
                     if util::has_flag(mouse.usButtonFlags, winuser::RI_MOUSE_WHEEL) {
-                        let delta = mouse.usButtonData as SHORT / winuser::WHEEL_DELTA;
+                        let delta =
+                            mouse.usButtonData as SHORT as f32 / winuser::WHEEL_DELTA as f32;
                         subclass_input.send_event(Event::DeviceEvent {
                             device_id,
                             event: MouseWheel {
```

Comment:
minor: the cast to `f32` should be removed now in `delta: LineDelta(0.0, delta as f32)`